cmd/zrobot

```diff
@@ -23,4 +23,12 @@ entry_point.add_command(blueprint)
 entry_point.add_command(godtoken)
 
 if __name__ == "__main__":
+    from jumpscale import j
+    if  j.core.state.configGetFromDict("myconfig", "backend") == "db":
```

Comment:
Why is this logic have to be put here ? Seems like the config manager itself could check on ` j.core.state.configGetFromDict("myconfig", "namespace", "")` directly so it's transparent for any user of the config manager

Reply:
Zdb Configmanager doesn't have to use a certain namespace for just 1 application in this case 0-robot, it sets the namespace in memory, allowing other applications to have their own namespaces while execution as specified here https://github.com/threefoldtech/jumpscale_core/issues/25#issuecomment-423941497